pkgs/development/python-modules/coinmetrics-api-client/default.nix

```diff
@@ -38,18 +46,27 @@ buildPythonPackage rec {
   ];
 
   nativeCheckInputs = [
+    mypy
```

Comment:
We don't really want to run linting tools in checkPhase

Reply:
Which phase do you believe is most appropriate?

Reply:
After consulting with @mweinelt, I've removed `mypy` entirely.

Reply:
In unsubscribed and you resubscribed me :cry: